web-marketplace/src/pages/BuyCredits/BuyCredits.Form.tsx

```diff
@@ -278,6 +287,24 @@ export const BuyCreditsForm = ({
     ],
   );
 
+  const stripeAmount =
+    data?.[CURRENCY_AMOUNT] ||
+    getCurrencyAmount({
+      currentCreditsAmount: 1,
+      card: paymentOption === PAYMENT_OPTIONS.CARD,
+      orderedSellOrders: cardSellOrders,
+      creditTypePrecision: creditTypeData?.creditType?.precision,
+    }).currencyAmount;
```

Comment:
should we put this in a `useMemo` too? `getCurrencyAmount` is a bit computation heavy